mmedit/models/backbones/encoder_decoders/decoders/aot_decoder.py

```diff
@@ -0,0 +1,49 @@
+import torch.nn as nn
+import torch.nn.functional as F
+from mmcv.cnn import ConvModule
+
+from mmedit.models.registry import COMPONENTS
+
+
+@COMPONENTS.register_module()
+class AOTDecoder(nn.Module):
+    """Decoder used in AOT-GAN model.
+    This implementation follows:
+    Aggregated Contextual Transformations for High-Resolution Image Inpainting
+    Args:
+        in_channels (int): Channel number of input feature.
+        act_cfg (dict): Config dict for activation layer, "relu" by default.
+    """
+
+    def __init__(self, in_channels=256, act_cfg=dict(type='ReLU')):
+        super().__init__()
+
+        self.dec1 = ConvModule(
+            in_channels,
+            128,
+            kernel_size=3,
+            stride=1,
+            padding=1,
+            act_cfg=act_cfg)
+        self.dec2 = ConvModule(
+            128, 64, kernel_size=3, stride=1, padding=1, act_cfg=act_cfg)
+        self.dec3 = ConvModule(
+            64, 3, kernel_size=3, stride=1, padding=1, act_cfg=None)
+
+        self.output_act = nn.Tanh()
+
+    def forward(self, x):
+        """Forward Function.
+        Args:
+            x (torch.Tensor): Input tensor with shape of (n, c, h, w).
+        Returns:
+            torch.Tensor: Output tensor with shape of (n, c, h', w').
+        """
+        for i in range(3):
+            if i <= 1:
+                x = F.interpolate(
+                    x, scale_factor=2, mode='bilinear', align_corners=True)
+            x = getattr(self, f'dec{i + 1}')(x)
+        x = self.output_act(x)
+
+        return x
```

Comment:
One suggestion: We can use `nn.ModuleList` to wrap the three decoders. During initialization, we have 
```
self.decoder = nn.ModuleList(ConvModule(...), ConvModule(...), ConvModule(...))
```
Then here we can simply do 
```
for i in range(0, len(self.decoder)):
  if i <= 1:
      x = F.interpolate(
          x, scale_factor=2, mode='bilinear', align_corners=True)
  x = self.decoder[i](x)

return self.output_act(x)
```

Reply:
@ckkelvinchan thank you for your suggestion! I originally wrote it without wrapping the three decoders in order to be consistent with the style of other decoders in mmediting (see gl_decoder.py for example). Just wondering if I should keep that or change as u suggested..

Reply:
I think if the name does not matter, we can simplify it. I would prefer using `nn.ModuleList`.